path_static_roles.go

```diff
@@ -441,9 +458,19 @@ type staticAccount struct {
 // NextRotationTime calculates the next rotation by adding the Rotation Period
 // to the last known vault rotation
 func (s *staticAccount) NextRotationTime() time.Time {
+	// LastVaultRotation can be zero if we skipped import rotation
+	if s.LastVaultRotation.IsZero() {
```

Comment:
Should we instead set `s.NextVaultRotation` when `!s.LastVaultRotation.IsZero()` and then always return `s.NextVaultRotation`?

Reply:
I considered that initially. After more thought, I think we should do that to keep it easier to reason about.

Reply:
Done in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/140/commits/988a9bee5d8192ff302e0b30762b174dd1d460a4

Reply:
No need to check for the zero value anymore since we make sure to set NextVaultRotation when we are setting LastVaultRotation.